server/apps/publish/formatters/field_mappers/selectorcode_mapper_tests.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.tests import TestCase
+from apps.publish import init_app
+from apps.publish.formatters.field_mappers.selectorcode_mapper import SelectorcodeMapper
+
+
+class SelectorcodeMapperTest(TestCase):
+
+    desks = [{'_id': 1, 'name': 'National'},
+             {'_id': 2, 'name': 'Sports'},
+             {'_id': 3, 'name': 'Finance'}]
+
+    def setUp(self):
+        super().setUp()
+        with self.app.app_context():
+            self.app.data.insert('desks', self.desks)
+            init_app(self.app)
+
```

Comment:
Have a look at the file and let me know if you see any other issues. Your comment missed the line I guess.